.changeset/witty-suits-worry.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fluidframework/tree": minor
+---
+
+Expose experimental alpha APIs for producing schema from enums
+
+`adaptEnum` and `enumFromStrings` have been added to `@fluidframework/tree/alpha`.
+These are unstable alpha APIs, are relatively simple helpers on-top of public APIs (source: [schemaCreationUtilities.ts](https://github.com/microsoft/FluidFramework/blob/a97d2a19bc063e15be38395154ddbc7b3ddca50f/packages/dds/tree/src/simple-tree/schemaCreationUtilities.ts)):
```

Comment:
Do you want to link to a specific commit or to main?

Reply:
Linking main was intentional. Neither option is great though. As these are alpha APIs, I figured linking main would be better for people looking for the latest code to use an example. That has the risk of becoming a stale link though. As this is just unstable/alpha stuff having the link break later doesn't seem too bad.

Reply:
```suggestion
These are unstable alpha APIs, are relatively simple helpers on-top of public APIs (source: [schemaCreationUtilities.ts](https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/src/simple-tree/schemaCreationUtilities.ts)):
```

If you want to link to main you need to update the link.

Reply:
Ops. Great catch.